docs/index.adoc

```diff
@@ -37,7 +37,14 @@ include::installing-korrel8r-ocp.adoc[leveloffset=+1]
 include::using-korrel8r.adoc[leveloffset=+1]
 
 [id="index-api-reference"]
-== API reference documentation
+// REST API
+include::zz_rest_api.adoc[leveloffset=+1]
+
+
+[id="index-domain-reference"]
+== Domain Reference
 
 include::zz_domains.adoc[leveloffset=+2]
```

Comment:
```suggestion
[id="index-domain-reference"]
== Domain reference

include::zz_domains.adoc[leveloffset=+2]
```
Might be better to just leave the heading out here and have a heading in the .adoc instead, but I'd need to look at a rendered preview first to see how this looks. Kinda reviewing blind atm.

Reply:
`make _site` then open `_site/index.html` - that's the same rendering used on the github.io site.
The domains are not exactly an API (and not exactly NOT an API) so I gave them their own heading rather than being part of "API reference". There is inconsistency about generated files including or not-including their top level headings. For domains that's in our control entirely, for the REST API it's somewhat imposed by the tool I'm using to generate (which sucks and generates text with nasty grammar and all sorts, but haven't take time yet to fix it or find a better one) If you know of good REST API doc tools we can swap it. I will have an alergic reaction if it requires us to install a JVM, but maybe I can take a pill if it's really really good

.

Reply:
Is there a convention about headings being inside/outside included files? Inside makes the include more complete, but outside lets you make top level headings consistent. E.g. the REST API used to be titled "Korrel8r REST API" which made sense when it was a separate doc, but when it's included in the Korrel8r book it's seems like stutter to have chapters called "Korrle8r this", "Korrel8r that"...